src/Directory.Build.props

```diff
@@ -23,6 +23,16 @@
     <_FriendAssembliesPublicKey>PublicKey=0024000004800000940000000602000000240000525341310004000001000100352162dbf27be78fc45136884b8f324aa9f1dfc928c96c24704bf1df1a8779b2f26c760ed8321eca5b95ea6bd9bb60cd025b300f73bd1f4ae1ee6e281f85c527fa013ab5cb2c3fc7a1cbef7f9bf0c9014152e6a21f6e0ac6a371f8b45c6d7139c9119df9eeecf1cf59063545bb7c07437b1bc12be2c57d108d72d6c27176fbb8</_FriendAssembliesPublicKey>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <!--
+      GraphQL library is not compiled for EOL target frameworks, however tests and benchmarks are running against as many target frameworks (even EOL) as possible
+      Hide EOL warnings (e.g. net5):
+      "warning NETSDK1138: The target framework 'net5' is out of support and will not receive security updates in the future.
+      Please refer to https://aka.ms/dotnet-core-support for more information about the support policy."
+    -->
```

Comment:
```suggestion
```

I would remove this comment; it is false.  For example, GraphQL.SystemTextJson is also compiled for .NET 5, and GraphQL.Server is also compiled for .NET Core 2.1.

Besides, the comment should be unnecessary -- the property name is self descriptive and can be easily looked up if not.

Reply:
Done